### PR TITLE
Changelog v1.0.0.alpha.1

### PR DESCRIPTION
**Pull request by dhavalshreyas**
Anything else we should call out here?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,21 @@
 Change Log
 ==========
 
+## Version 1.0.0.alpha.1
+
+_2020-07-07_
+
+* Breaking: `Worker`s have now moved from the core `Workflow` library to `WorkflowReactiveSwift`. (#12)
```
Comment (AquaGeek) [inline CHANGELOG.md:8]: Nit: "now" is unnecessary:

>`Worker`s have moved from…

We should also add the "fix-it" directions:

> You will need to add that as a dependency and import that module in any files with `Worker`s.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
+* Breaking: `Worker`s are now keyed based on type. ([workflow/1021](https://github.com/square/workflow/issues/1021))
```
Comment (AquaGeek) [inline CHANGELOG.md:9]: Let's expand on what this means — what changes, how does this affect me, etc

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,21 @@
...
+* `Swift` and [`Kotlin`](https://github.com/square/workflow-kotlin) libraries now live in seperate repos.
+* `Worker`s are now built on a more fundamental concept called `SideEffect`.  ([workflow/1021](https://github.com/square/workflow/issues/1021))
+* Introduce `WorkflowRxSwift` to build a `RxSwift` `Worker`. (#27)
+* Several APIs have been updated to be consistent. (#9)
+  * `MyWorkflow().rendered(with: context)` to `MyWorkflow().rendered(in: context)`
+  * `context.awaitResult(for: MyWorker())` to `MyWorker().running(in: context)`
```
Comment (AquaGeek) [inline CHANGELOG.md:15]: We should mention that the old versions are still around but are deprecated and will be removed for the beta.

**Comment by zach-klippenstein** [comment]
Version number should be 1.0.0-alpha.1 (semver.org says pre-release identifiers must be separated by a dash, and optionally followed by more version numbers separated by dots).

**Comment by AquaGeek** [comment]
> Version number should be 1.0.0-alpha.1 (semver.org says pre-release identifiers must be separated by a dash, and optionally followed by more version numbers separated by dots).

CocoaPods seems to follow a [different convention](https://guides.cocoapods.org/making/making-a-cocoapod.html#cocoapods-versioning-specifics), but I'd defer to @amorde or somebody with more CocoaPods knowledge.

**Comment by zach-klippenstein** [comment]
Is it just a convention? Google also doesn't follow semver rules with a lot of their artifacts, but that's no reason for us not to.

File: WorkflowReactiveSwift/Tests/SignalTests.swift [hunk 4]
```diff
@@ -85,6 +85,8 @@ class SignalTests: XCTestCase {
         host.update(workflow: SignalTestWorkflow(signal: signalTwo))
         wait(for: [expectation], timeout: 1)
     }
+
+    func test_deprecated_signalWorker() {}
```
Comment (bencochran) [inline WorkflowReactiveSwift/Tests/SignalTests.swift:89]: 👀

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,6 +1,23 @@
...
+  * You will need to add that as a dependency and import that module in any files with `Worker`s.
+* Breaking: `Worker`s are now keyed based on type. ([workflow/1021](https://github.com/square/workflow/issues/1021))
+  * If you were running multiple `Worker`s of the same type on a `render` call, please use a `key` while running it: `MyWorker().running(in: context, key: "key")`
+* `Swift` and [`Kotlin`](https://github.com/square/workflow-kotlin) libraries now live in seperate repos.
+* `Worker`s are now built on a more fundamental concept called `SideEffect`.  ([workflow/1021](https://github.com/square/workflow/issues/1021))
+* Introduce `WorkflowRxSwift` to build a `RxSwift` `Worker`. (#27)
+* Several APIs have been updated to be consistent. The older APIs have been deprecated and will be removed from the `beta`. (#9)
+  * `MyWorkflow().rendered(with: context)` to `MyWorkflow().rendered(in: context)`
+  * `context.awaitResult(for: MyWorker())` to `MyWorker().running(in: context)`
+* Introduce new `RenderTester` API. (#15)
```
Comment (bencochran) [inline CHANGELOG.md:18]: Maybe also mention here that the old APIs will be removed in beta

File: WorkflowReactiveSwift/Tests/SignalTests.swift (hunk 4 above)
Comment (dhavalshreyas) [reply WorkflowReactiveSwift/Tests/SignalTests.swift:89]: oops

**Comment by dhavalshreyas** [comment]
> > Version number should be 1.0.0-alpha.1 (semver.org says pre-release identifiers must be separated by a dash, and optionally followed by more version numbers separated by dots).
> 
> CocoaPods seems to follow a [different convention](https://guides.cocoapods.org/making/making-a-cocoapod.html#cocoapods-versioning-specifics), but I'd defer to @amorde or somebody with more CocoaPods knowledge.

That's weird. Cocoapods seems to [recommend](https://guides.cocoapods.org/making/specs-and-specs-repo.html) Semantic Versioning. `1.0.0-alpha.1` is SemVer compliant.

**Comment by amorde** [comment]
I think the CocoaPods docs that were linked are there for example, not an explicit format requirement. `1.0.0-alpha.1` should work with CocoaPods AFAIK 👍

**Pull request merged**